src/HasSuffixCollection.php

```
@@ -225,16 +223,15 @@ public function suffixSetAttribute(string|UnitEnum $key, mixed $value, string $p
+     * $component->suffixTag(Inline::SPAN);
      * $component->suffixTag(false);
      * ```
      *
-     * @param BlockInterface|false|InlineInterface|VoidInterface $value Tag name for the suffix element, or `false` to
-     * disable.
+     * @param false|UnitEnum $value Tag name for the suffix element, or `false` to disable.
      *
      * @return static New instance with the updated `suffixTag` value.
      */
-    public function suffixTag(false|BlockInterface|InlineInterface|VoidInterface $value = false): static
+    public function suffixTag(false|UnitEnum $value = false): static
```

PHPMD (2.15.0)

[error] The method suffixTag has a boolean flag argument $value, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)